### ISSUE TITLE
Aspect of the Viper yields threat. 

### ISSUE THREAD

**Issue by PekelZoda** [SunstriderEmu/BugTracker#2341]
**Describe the bug**

For each tick of mana you gain with AotV, mobs gain threat and aggro you. This does not happen with normal spirit regen, nor when you drink water during a fight. Also, EVEN if your mana bar is full, you will still gain threat from using AotV, while gaining no mana. 

**To Reproduce**

1. Send your pet into a mob with neighbouring mobs so that it aggroes more than one. 
2. Let the pet fight the 1 mob for a while, and simply get hit by the others.
3. Turn on Aspect of the Viper.
4. At the first tick of your mana, the neighbouring mobs will aggro you. 

Creature: https://tbc-twinhead.twinstar.cz/?npc=8912  (and more)
Spell: https://tbc-twinhead.twinstar.cz/?spell=34074

**Expected behavior**

Aspect of the Viper would give no different effects to threat than normal spirit regen would. That is, it should yield zero threat, so mobs should stay on your pet, and would not aggro the hunter. 

Thank you very much, dear sirs!

**Comment by Felynda** [comment]
Not bugged
https://classic.wowhead.com/guides/threat-overview-classic-wow
"Mana generates threat at a 1-to-0.5 ratio (100 mana gained = 50 threat generated), Rage generates threat at a 5-to-1 ratio (50 rage = 250 threat), and Energy generates threat at a 5-to-1 ratio as well. The threat generated by these resources is distributed equally to all of the mobs that are actively engaged by the party/raid. In addition, these threat values are static and cannot be altered by threat modifiers (talents, abilities, etc)."

IF you are at full mana this should do nothing though. This might be bug.

**Comment by PekelZoda** [comment]
Good point, Felynda. Thank you for your answer!

My point, however, is that AotV is to be considered a 'natural' regeneration of mana, and should therefore not generate threat. 

Quoting the paragraph above the one that you quoted: 
https://classic.wowhead.com/guides/threat-overview-classic-wow
"Mana, Rage, Energy Gains: Gaining resources while in combat with a mob will add threat to the threat table, unless the resource is gained naturally. For example, if a  Rogue gains 20 energy through a normal tick that occurs every x seconds, it would not generate threat. Similarly, a  Priest gaining mana by means of regenerating naturally would not generate threat. Talents, abilities and items that grant Mana, Energy, or Rage will generate threat (for example,  Major Mana Potion and  Relentless Strikes). To determine if threat is generated, check the combat log for any "you gain x rage", "you gain x mana", etc lines."

Again, AotV has always been considered a boost to your natural mana regen, and has never yielded threat, so it is still a bug. 

I've been checking my combat log, btw, and I see no logs on mana gain from AotV. (I do see mana gains from mana potions). 

(I tested drinking mana pot in combat, and it yielded threat, so at least that works)

**Comment by Jerrythemagicdog** [comment]
> Again, AotV has always been considered a boost to your natural mana regen, and has not yielded threat in retail, so it is still a bug.

AotV has a passive aura running in the background granting you mana every 5 seconds and does not increase your base mana regen which is the correct retail behavior.

**Comment by PekelZoda** [comment]
Fine. However: it should not genererate threat.

**Comment by Jerrythemagicdog** [comment]
Thanks, could you see if the issue is resolved after next server update (most likely in 24hrs from now)?

**Comment by PekelZoda** [comment]
Ye, I'll keep a look out for sure! Thanks!

**Comment by PekelZoda** [comment]
I tested it just now, post server update, and the bug is still there. 

I went into the Stockades and sent my pet into a gang of mobs, and watched. When I was NOT using AotV, every mob kept attacking my pet. 

When I was using AotV, the untouched mobs turned and attacked me (the hunter) after a single tick up of AotV. 

This happened regardless of my mana bar being full, or if I actually was regenerating some mana. 

All in all: I can observe *no change* since the server update. 

Still, I appreciate your attention.
